app/models/widget/card.rb

```diff
@@ -9,6 +9,8 @@ class Widget::Card < ApplicationRecord
   include Globalizable
 
   validates_translation :title, presence: true
+  validates :link_url, presence: true,
+            if: -> { !header? || (header? && link_text.present?) }
```

Comment:
```suggestion
  validates :link_url, presence: true, if: -> { !header? || link_text.present? }
```

When `!header?` is false, `header?` will always be true, and so including it in the secod condition is redundant.

Reply:
Does it make sense to add an additional rule? :thinking:

```ruby
validates :link_text, presence: true, if: -> { header && link_url.present? }
```